### ISSUE TITLE
install-required-packages.sh and CentOS 8

### ISSUE THREAD

**Issue by thiagoftsm** [netdata/netdata#9380]
##### Bug report summary
Today I had to install on CentOS 8 a Netdata client and I decided to use `install-required-packages.sh`, I observed that at the end the packages installed did not allow me to use the `dbengine`, because we are missing packages and the libuv installed is not recognized. I had to run the following commands to be able to compile Netdata:

```bash
# ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive
# dnf remove -y libuv1 
# dnf -y install libuv-devel zlib-devel libuuid-devel 
# ./netdata-installer.sh --dont-wait 
```

##### OS / Environment
```
Linux malta2183.startdedicated.com 4.18.0-193.6.3.el8_2.x86_64 #1 SMP Wed Jun 10 11:09:32 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
/etc/centos-release:CentOS Linux release 8.2.2004 (Core) 
/etc/os-release:NAME="CentOS Linux"
/etc/os-release:VERSION="8 (Core)"
/etc/os-release:ID="centos"
/etc/os-release:ID_LIKE="rhel fedora"
/etc/os-release:VERSION_ID="8"
/etc/os-release:PLATFORM_ID="platform:el8"
/etc/os-release:PRETTY_NAME="CentOS Linux 8 (Core)"
/etc/os-release:ANSI_COLOR="0;31"
/etc/os-release:CPE_NAME="cpe:/o:centos:centos:8"
/etc/os-release:HOME_URL="https://www.centos.org/"
/etc/os-release:BUG_REPORT_URL="https://bugs.centos.org/"
/etc/os-release:
/etc/os-release:CENTOS_MANTISBT_PROJECT="CentOS-8"
/etc/os-release:CENTOS_MANTISBT_PROJECT_VERSION="8"
/etc/os-release:REDHAT_SUPPORT_PRODUCT="centos"
/etc/os-release:REDHAT_SUPPORT_PRODUCT_VERSION="8"
/etc/os-release:
/etc/redhat-release:CentOS Linux release 8.2.2004 (Core) 
/etc/system-release:CentOS Linux release 8.2.2004 (Core) 
```

##### Netdata version
netdata v1.22.1-297-g13624215
 

##### Component Name
Installer

##### Steps To Reproduce
1. Install CentOS 8 with the minimum possible
2.Run the commands described on top.

##### Expected behavior
After to use `install-required-packages.sh`, it is expected to have conditions to compile Netdata with `dbengine`.

**Comment by thiagoftsm** [comment]
We also need to install `openssl-devel` and `cmake`.

**Comment by Ferroin** [comment]
Known and expected behavior when running `install-required-packages.sh` without specifying a package set to install. It ideally should complain instead of continuing the way it does right now, but this is not causing problems for users who are using the `kickstart.sh` install script or following our installation documentation correctly.

@thiagoftsm for future reference, the required command is `./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata` to explicitly pull in all the build dependencies.

**Comment by thiagoftsm** [comment]
> Known and expected behavior when running `install-required-packages.sh` without specifying a package set to install. It ideally should complain instead of continuing the way it does right now, but this is not causing problems for users who are using the `kickstart.sh` install script or following our installation documentation correctly.
> 
> @thiagoftsm for future reference, the required command is `./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata` to explicitly pull in all the build dependencies.

I used `./packaging/installer/install-required-packages.sh --dont-wait` when I had the problem reported.

**Comment by Ferroin** [comment]
> > Known and expected behavior when running `install-required-packages.sh` without specifying a package set to install. It ideally should complain instead of continuing the way it does right now, but this is not causing problems for users who are using the `kickstart.sh` install script or following our installation documentation correctly.
> > @thiagoftsm for future reference, the required command is `./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata` to explicitly pull in all the build dependencies.
> 
> I used `./packaging/installer/install-required-packages.sh --dont-wait` when I had the reported problem.

That's my point though, you didn't list which set of packages to pull in at the end, and therefore didn't get most of the packages. Our docs tell people (in an admitedly roundabout way) to specify either `netdata` or `netdata-all` in the command, and we explicitly use `netdata` for the regular install process.

**Comment by thiagoftsm** [comment]
I will try again, but considering that I could move forward and at the end I did not have what I was expecting ( Netdata + dbengine). Could we consider this a bug?

**Comment by Ferroin** [comment]
Yes, I would consider the current behavior to be a bug, but I would not consider it a particularly high priority given that:

1. Assuming users are following our documentation, they shouldn't run into it.
2. We may actually end up eliminating this script as part of the upcoming packaging rework.

**Comment by prologic** [comment]
I have fixed this with #9399 which now makes `netdata-all` the default behaviour.

**Comment by Ferroin** [comment]
Closing as `wontfix` based on the reasoning in https://github.com/netdata/netdata/issues/9380#issuecomment-647702190.
